mne/io/meas_info.py

```diff
@@ -56,6 +56,8 @@
     ecog=(FIFF.FIFFV_ECOG_CH, FIFF.FIFFV_COIL_EEG, FIFF.FIFF_UNIT_V),
     fnirs_cw_amplitude=(FIFF.FIFFV_FNIRS_CH,
                         FIFF.FIFFV_COIL_FNIRS_CW_AMPLITUDE, FIFF.FIFF_UNIT_V),
+    fnirs_fd_phase=(FIFF.FIFFV_FNIRS_CH, FIFF.FIFFV_COIL_FNIRS_FD_PHASE,
+                    FIFF.FIFF_UNIT_V),
```

Comment:
We probably want to store the data in radians since it's SI, but display in degrees. So this should be `FIFF.FIFF_UNIT_RAD`, and the `raw.get_data()` should return radians, but plotting functions should always show degrees. Make sense?

Also our SI test will need to be updated below, too.

If it's not clear how to move this direction I can take a stab at it next week.